guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,91 @@
+[id="Renewing_a_Custom_SSL_Certificate_Proxy_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+Do not use the same command on more than one {SmartProxyServer}.
+
+.Prerequisites
+Before renewing a custom server certificate for {SmartProxyServer}, ensure that your {Project} and {SmartProxies} meet the following conditions:
+
+* {ProjectServer} is configured with a custom certificate.
+* {SmartProxyServer} is registered to {ProjectServer}.
+* The {SmartProxyServer} packages are installed.
+
+.Procedure
+You must create a new Certificate Signing Request and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {SmartProxyServer} certificate and CA bundle.
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a folder and verifying them through the `katello-certs-check` command.
+
+To renew a {customssl} certificate for your {SmartProxyServer}, complete the following steps:
+
+. On {ProjectServer} validate the {customssl} certificate input files:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-c _/root/capsule_cert/New_capsule_cert.pem_ \
+-k _/root/capsule_cert/capsule_cert_key.pem_ \
+-b _/root/capsule_cert/ca_cert_bundle.pem_
+----
++
+If you set a wildcard value `*` for the certificate’s Common Name `CN =` in the `/root/capsule_cert/openssl.cnf` configuration file, you must add the `-t capsule` option to the `katello-certs-check` command.
```

Comment:
for capsule, you can generally use "smart-proxy-context" for branding and to prevent whitespace: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L95